impl/src/bitfield/analyse.rs

```diff
@@ -105,6 +105,9 @@ impl BitfieldStruct {
             let path = &meta.path;
             if path.is_ident("Debug") {
                 config.derive_debug(path.span())?;
+            } else if path.is_ident("uDebug") {
+                #[cfg(feature = "ufmt")]
```

Comment:
There should be no match at all on `uDebug` if this feature is disabled, rather than silently consumed